fda-model/stageDamage/ImpactAreaStageDamage.cs

```diff
@@ -87,7 +87,7 @@ private void AddRules()
         {
             AddSinglePropertyRule(nameof(_inventory), new Rule(() => { _inventory.Validate(); return !_inventory.HasErrors; }, $"The structure inventory has errors: " + _inventory.GetErrors().ToString(), _inventory.ErrorLevel));
             AddSinglePropertyRule(nameof(convergenceCriteria), new Rule(() => { convergenceCriteria.Validate(); return !convergenceCriteria.HasErrors; }, $"Convergence criteria has errors: " + convergenceCriteria.GetErrors().ToString(), convergenceCriteria.ErrorLevel));
-            if(_AnalyticalFlowFrequency != null)
+            if (_AnalyticalFlowFrequency != null)
```

Comment:
We don't need this extra space.

Reply:
I did CTRL +K, CTRL + F on this class and that added the spaces

Reply:
Ok, i was just talking about the space after the "if". That would be weird if the auto format did that.